### PR TITLE
Adding Medo Script command

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description

Script command that lets user add tasks, toggle floating window using Raycast!
A companion to https://github.com/Aayush9029/Medo 

## Type of change

- [x] New script command

## Screenshot
CleanShot 2022-03-24 at 22 45 25@2x

## Dependencies / Requirements
None

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/medo/add-task.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```
Comment (dehesa) [inline commands/apps/medo/add-task.sh:2]: Please add a dependency note as required by the [CONTRIBUTING.md](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-apps).

File: commands/apps/medo/add-task.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Add Task
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/medo.png
+# @raycast.argument1 { "type": "text", "placeholder": "Title" }
+# @raycast.argument2 { "type": "text", "placeholder": "priority", "default": "low","optional": true}
```
Comment (dehesa) [inline commands/apps/medo/add-task.sh:11]: `priority` should have the first letter uppercased. Therefore `Priority` 🙂

File: commands/apps/medo/add-task.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# @raycast.packageName Medo
+
+# Documentation:
+# @raycast.description Add a new task
+# @raycast.author Aayush 
+# @raycast.authorURL https://github.com/Aayush9029/Medo
```
Comment (dehesa) [inline commands/apps/medo/add-task.sh:17]: This should be the author's URL, not the project's URL.

**Review by dehesa — CHANGES_REQUESTED** [review]
Hi @Aayush9029 , thank you for the contribution. I left you some comments below that should be applied to all scripts 😉

**Comment by Aayush9029** [comment]
@dehesa lmk if I missed anything!

**Comment by dehesa** [comment]
That looks great! Merging...

**Pull request merged**